packages/core/src/components/Alert/Alert.scss

```diff
@@ -71,27 +86,37 @@ $alert-icon-size: $spacer-5;
 .ds-c-alert__text {
   margin-bottom: 0;
   margin-top: 0;
+  &:only-child {
+    margin-bottom: $spacer-2;
+    padding-top: $spacer-1;
+  }
 }
 
 .ds-c-alert--error {
   background-color: $color-error-lightest;
   background-image: url('#{$image-path}/error.png');
   background-image: url('#{$image-path}/error.svg');
-  border-color: $color-error-light;
+  &:before {
+    background-color: $color-secondary;
```

Comment:
For the "error", "warn", and "success" modifiers, we should use their corresponding Sass variables: `$color-error`, `$color-warn`, `$color-success`. This would correspond to people's expectations if they override those variables.